public/components/content-list-item/_content-list-item.scss

```diff
@@ -380,7 +380,7 @@
     .content-list-item__field--working-title,
     .content-list-item__field--headline {
         @include smallScreen {
-            @extend %fs-data-1;
+            @include font-size(11, 14); // Can't extend placeholders (%fs-data-1) into @media so including the font-size mixin directly
```

Comment:
I reaaaaaallly hate `@extend` anyway. Not one for now but we should look at getting rid of them.